runhouse/rns/folders/folder.py

```diff
@@ -678,9 +762,16 @@ def put(self, contents, overwrite=False):
             with fss_file as f:
                 f.write(raw_file)
 
+    @staticmethod
+    def bucket_name_from_url(url: str) -> str:
+        """Extract the bucket name from a URL (e.g. '/my-bucket/my-folder/my-file.txt' -> 'my-bucket')"""
+        split_urls: list = list(filter(None, url.split('/')))
```

Comment:
fyi `.split('/')` may not always work, if `url` can refer to a local path on Windows (which uses forward slash `\`)